api/services/managedInstance/aws/responses/bulk-migration.yaml

```diff
@@ -0,0 +1,23 @@
+description: Bulk Migration Response
+content:
+  application/json:
+    schema:
+      allOf:
+        - $ref: "../../../../commons/schemas/responseItemWrapper.yaml"
+        - type: "object"
+          properties:
+            response:
+              type: "object"
+              properties:
+                items:
+                  type: "array"
+                  items:
+                    $ref: "../schemas/bulk-migration.yaml"
+                failures:
+                  type: "array"
+                  title: Failed instances
+                  description: AWS EC2 Instances that we failed to start migrating. In case there are no failures, would return empty list
```

Comment:
```suggestion
                  description: AWS EC2 instances that we failed to start migrating. In case there are no failures, would return empty list
```